app/components/AccountSetting/AccountSettingSection.js

```diff
@@ -18,14 +30,18 @@ export default class AccountSettingSection extends Component {
     });
   }
 
+  handleEmailChange(event) {
+    this.setState({ email: event.target.value});
+  }
+
   render() {
-    const { email, children } = this.props;
+    const { location, email, children } = this.props;
 
     return (
       <div className='account-setting'>
         <Header headerTitle='Account Settings'/>
         <div className='container' style={ { width: 300 } }>
-          <EmailInput email={ email } handleBlur={ this.navigateToPage.bind(null, '/AccountSetting/NewEmail') }/>
+          <EmailInput onEmailChange={ this.handleEmailChange } initialEmail={ email } currentEmail={ this.state.email } handleBlur={ this.navigateToPage.bind(null, '/AccountSetting/NewEmail') }/>
```

Comment:
I need initialEmail to compare it with currentEmail in handleBlur(event) in  EmailInput.js